app/models/concerns/thredded/post_common.rb

```diff
@@ -13,14 +13,18 @@ module PostCommon
 
       validates :content, presence: true
 
-      scope :order_oldest_first, -> { order(id: :asc) }
-      scope :order_newest_first, -> { order(id: :desc) }
+      scope :order_oldest_first, -> { order(created_at: :asc) }
```

Comment:
If 10 posts are made within 5 minutes, the probability of there being at least two posts with exactly the same timestamp is 14%! Calculator: http://algoviz.org/OpenDSA/dev/OpenDSA/AV/Hashing/Birthday.html